### DISCUSSION TITLE
Converting JSON to HASH and Vice Versa

### DISCUSSION THREAD

**Discussion by wazini-john** [danitseitlin/redis-modules-sdk-ts#149] [Q&A]
How do I convert JSON to REDIS HASH and REDIS HASH TO JSON. Does the lib offer that ability?
Don't want to use redis-json currently

**Comment by cngeru** [comment]


**Reply by danitseitlin** [reply]
@wazini-john Hi, Sorry for the delayed reply. What do you mean by converting, meaning using the REDIS hash like a JS object?
i.e. for:
{
  obj: {
    key: 1
  }
}

You want to be able to do: get obj:key and get 1 as the returned value?

**Reply by wazini-john** [reply]
Wanted to use redisearch and needed the hash. Used the lib above. Got a query though https://github.com/danitseitlin/redis-modules-sdk-ts/discussions/163
